### DISCUSSION TITLE
TeslaMate Donate and update missing from home page

### DISCUSSION THREAD

**Discussion by cwanja** [teslamate-org/teslamate#1457] [Q&A]
Looking through the commits, I did not see anywhere it was removed. 

But from the home page, the "donate" and "update available" for TeslaMate specifically are missing.

**Comment by adriankumpf** [comment]
That's a feature!

The update check runs every three days (or a few minutes after restarting TeslaMate). Only when a new version is available, the icon is displayed. 

And the "Donate" button is displayed only for new users and after installing certain updates, to avoid being too much of an annoyance :) (the donate link is also shown on the GitHub project page under "Sponsor this project")
